modules/@apostrophecms/widget-type/ui/apos/mixins/AposWidgetMixin.js

```diff
@@ -12,8 +12,10 @@ export default {
   },
   watch: {
     modelValue: {
-      handler() {
-        this.renderContent();
+      handler(widget) {
+        if (widget.type !== '@apostrophecms/layout-column') {
+          this.renderContent();
+        }
```

Comment:
This fixes the issue of the rich text unfocusing because the entire area is re rendered through livre preview when typing (since before the rich text used it's own component without this mixin and now layout-widget and layout-column use this mixin with `AposWidget`).
I don't thing `layout-widget` and `layout-column-widget` should use `AposWidget` like for a regular widget. I thing they should have their own basic componnent (like we did for quick image upload).
Let me know what you think.

Reply:
I would not say that they are regular widget, they're pretty special in my opinion, they wrap normal widgets.
But I would be ok with some widget option to disable the live preview.